site/docs/pages/config/types.mdx

```diff
@@ -5,6 +5,20 @@ description: Glossary of OnchainKit top-level Types.
 
 # Types [Glossary of general Types.]
 
+## `AppConfig`
+
+```ts
+type AppConfig = {
+  appearance?: {
+    name?: string | null; // The name of your application
+    logo?: string | null; // The URL of your application logo
```

Comment:
nit: Add the Optionally set... prefix

Reply:
I'm of the opinion we should remove these from `theme` and `mode` - developers can infer this as optional from the `?` operator and the type, it's additional cognitive complexity for the developer to parse this additional text from the docstring for each individual field

Reply:
this is an onchainkitConfig paradigm.  I'm unsure why some items were implemented as value | null and some as value | undefined.